requirements.txt

```diff
@@ -2,16 +2,5 @@
 django>=1.8,<1.9
 django-model-utils==2.3.1
```

Comment:
Could I suggest you make a local_requirements.txt which are to be used for when running the repo as a stand-alone (i.e. with unit tests)? My concern is what happens if edx-platform - say - goes to 2.4.x of django-model-utils but the requirement is not updated in this repo. Seems like there could be a tug-of-war between competing requirement versions. Anyhow, I'm not sure if there's a edX best practice about that, but that was how I approached it in edx-proctoring.